dockerfiles/gw_docker_build_docs

```diff
@@ -0,0 +1,29 @@
+
+FROM ubuntu:20.04
+USER root
+
+RUN apt update -y && apt upgrade -y && \
+    apt install -y software-properties-common && \
+    add-apt-repository ppa:ubuntugis/ppa && \
+    apt update -y && apt install -y \
+    gdal-bin \
+    geotiff-bin \
+    git \
+    libgdal-dev \
+    libgl1 \
+    libspatialindex-dev \ 
+    wget \
+    python-is-python3 \
+    pip \
+    g++ 
+
+RUN pip install Cython numpy  
+RUN pip install git+https://github.com/jgrss/geowombat
+RUN pip install sklearn-xarray  pip-tools rtree ipykernel sphinx graphviz numpydoc sphinx_automodapi sphinxcontrib.bibtex==1.0
+
+RUN python -c "import geowombat as gw"
+RUN python -c "import rasterio;from rasterio.crs import CRS; cc = CRS.from_epsg(4326)"
```

Comment:
These are just tests, right?

Reply:
Yes that's just a basic test to make sure things are set up.